.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [22.x]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Cache Angular build
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.angular
+            .angular/cache
+          key: ${{ runner.os }}-angular-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-angular-
+      - name: Install dependencies
+        run: npm ci
+      - name: Run ESLint
+        run: npm run lint
+      - name: TypeScript compilation check
+        run: ng build --configuration development
```

Comment:
this line causes the problem in the process of CI.

the error message is below
```
Run ng build --configuration development
  ng build --configuration development
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/59[2](https://github.com/m-yoshiro/angular-todo-app/actions/runs/15747733837/job/44387082650?pr=9#step:7:2)1b238-d2b9-4f20-9a0e-160f7ba72ed4.sh: line 1: ng: command not found
Error: Process completed with exit code 127.
```